test/spec/videoCache_spec.js

```diff
@@ -48,9 +49,17 @@ describe('The video cache', () => {
       xhr = sinon.useFakeXMLHttpRequest();
       requests = [];
       xhr.onCreate = (request) => requests.push(request);
+      config.setConfig({
```

Comment:
Why wouldn't you just use `newConfig()` here? Like `dfpAdServerVideo_spec.js` does.

It cuts down on the changes you need to make for this PR by quite a bit.

Reply:
I don't think that `newConfig` is working as you are intending it to.  For instance, I just removed it from `dfpAdServerVideo_spec.js` file and the tests still pass.  The actual `dfpAdServerVideo.js` module is operating on the `config` object in in `config.js`, not the new config object created with `newConfig()`, so unless you change these modules to dependency injection where you inject the config object, using `newConfig` doesn't do anything.